src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -40,7 +44,35 @@
      * София - 900</p>
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-            throws Exception {
-        return null;
+            throws IllegalDateParametersException {
+        final String pattern = "dd.MM.yyyy";
```

Comment:
make it  constant

Reply:
done

Reply:
can we use const in java?

Reply:
https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java

Reply:
done